### PR TITLE
skip the cases when MCO was customized installation

### PR DESCRIPTION
**Pull request by hchenxa**
Signed-off-by: hchenxa <huichen@redhat.com>

1. if the MCO was created and already exists, no need to check the default value.
2. if the MCO CR did not have retentionConfig, update MCO will throws null point exception, so in this case, only update the storage size and test this part.

### CODE REVIEW

File: pkg/tests/observability_reconcile_test.go [hunk 1]
```diff
@@ -53,6 +53,16 @@ var _ = Describe("Observability:", func() {
...
+		mcoCR, err := dynClient.Resource(utils.NewMCOGVRV1BETA2()).Get(MCO_CR_NAME, metav1.GetOptions{})
+		Expect(err).NotTo(HaveOccurred())
+		if _, adv := mcoCR.Object["spec"].(map[string]interface{})["advanced"]; !adv {
+			Skip("Skip the case since the MCO CR did not have advanced spec configed")
+		} else {
+			if _, rec := mcoCR.Object["spec"].(map[string]interface{})["advanced"].(map[string]interface{})["retentionConfig"]; !rec {
+				Skip("Skip the case since the MCO CR did not have advanced retentionConfig configed")
+			}
+		}
+
```
Comment (clyang82) [inline pkg/tests/observability_reconcile_test.go:65]: suggest to have an util method to judge if the field in CR exists or not.

Comment (hchenxa) [reply pkg/tests/observability_reconcile_test.go:60]: updated

**Review by morvencao — COMMENTED** [review]
/lgtm

**Pull request merged**